data/translations.json

```diff
@@ -1,644 +1,768 @@
-      "DocumentTrackingCurrentReleaseDateLabel": "Aktuelles Veröffentlichungsdatum",
+      "DocumentTrackingCurrentReleaseDateTitle": "Aktuelles Veröffentlichungsdatum",
       "DocumentTrackingCurrentReleaseDateDescription": "Das Datum, an dem die aktuelle Version dieses Dokuments veröffentlicht wurde",
-      "DocumentTrackingGeneratorLabel": "Dokumentengenerator",
+      "DocumentTrackingGeneratorTitle": "Dokumentengenerator",
       "DocumentTrackingGeneratorDescription": "Ist ein Container, der alle Elemente enthält, die sich auf die Erstellung des Dokuments beziehen. Diese Elemente verweisen darauf, wann das Dokument tatsächlich erstellt wurde, einschließlich des Erstellungsdatums und der Entität, die es erstellt hat.",
-      "DocumentTrackingGeneratorDateLabel": "Datum der Dokumentenerstellung",
+      "DocumentTrackingGeneratorDateTitle": "Datum der Dokumentenerstellung",
       "DocumentTrackingGeneratorDateDescription": "Dies SOLLTE das aktuelle Datum sein, an dem das Dokument erstellt wurde. Da Dokumente oft intern von einem Dokumentenproduzenten erstellt werden und eine von Null abweichende Zeitspanne existieren, bevor sie freigegeben werden, KANN sich dieses Feld von dem Datum der ursprünglichen Freigabe und dem aktuellen Freigabedatum unterscheiden.",
-      "DocumentTrackingGeneratorEngineLabel": "Dokumentenerstellungsengine",
+      "DocumentTrackingGeneratorEngineTitle": "Dokumentenerstellungsengine",
       "DocumentTrackingGeneratorEngineDescription": "Enthält Informationen über die Engine, die das CSAF-Dokument erzeugt hat.",
-      "DocumentTrackingGeneratorEngineNameLabel": "Name der Engine",
+      "DocumentTrackingGeneratorEngineNameTitle": "Name der Engine",
       "DocumentTrackingGeneratorEngineNameDescription": "Stellt den Namen der Engine dar, die das CSAF-Dokument erzeugt hat.",
-      "DocumentTrackingGeneratorEngineVersionLabel": "Version der Engine",
+      "DocumentTrackingGeneratorEngineVersionTitle": "Version der Engine",
       "DocumentTrackingGeneratorEngineVersionDescription": "Enthält die Version der Engine, die das CSAF-Dokument erzeugt hat.",
-      "DocumentTrackingIdLabel": "Eindeutiger Bezeichner für das Dokument",
+      "DocumentTrackingIdTitle": "Eindeutiger Bezeichner für das Dokument",
       "DocumentTrackingIdDescription": "Die ID ist eine einfache Kennzeichnung, die eine breite Palette von Nummerierungswerten, -typen und -schemata zulässt. Ihr Wert SOLLTE von der Behörde, die das Originaldokument ausstellt, zugewiesen und gepflegt werden.",
-      "DocumentTrackingInitialReleaseDateLabel": "Datum der Erstveröffentlichung",
+      "DocumentTrackingInitialReleaseDateTitle": "Datum der Erstveröffentlichung",
       "DocumentTrackingInitialReleaseDateDescription": "Das Datum, an dem dieses Dokument erstmals veröffentlicht wurde.",
-      "DocumentTrackingRevisionHistoryLabel": "Revisionshistorie",
+      "DocumentTrackingRevisionHistoryTitle": "Revisionshistorie",
```

Comment:
Usually, in German we use: "Version" instead of "Revision"

Reply:
Please see #263